tic_tac_toe/board.py

```diff
@@ -0,0 +1,60 @@
+class Board(object):
+    def __init__(self, rows=3, cols=3):
+        self._board = [[0] * cols] * rows
```

Comment:
nên đổi thành self._board = [[0 for col in range(cols)] for row in range(rows)] thì khi cập nhật board chỉ cần self._board[x][y] = token

Reply:
cách này ok nha